lib/vm.js

```diff
@@ -18,19 +18,21 @@ const realRunInContext = Script.prototype.runInContext;
 
 Script.prototype.runInThisContext = function(options) {
   if (options && options.breakOnSigint) {
-    return sigintHandlersWrap(() => {
+    const sigintHandler = () => {
```

Comment:
If line length is below 80 chars I think it is ok to put the whole function expression on one line.